### PR TITLE
Add %find_lang macro to debbuild

### PR DESCRIPTION
**Pull request by Conan-Kudo**
This PR implements the `%find_lang` macro for debbuild so that we can create file list files for locale content to append to `%files` sections as is typically done for RPMs.

### CODE REVIEW

**Comment by Conan-Kudo** [comment]
@daxelrod-datto So far, this only includes the import from Mageia, but modifications definitely need to be made before the `%find_lang` macro itself can be added.

File: cpanfile [hunk 1]
```diff
@@ -3,4 +3,5 @@ requires 'Locale::gettext', '1.0';
 on test => sub {
   requires 'Test2::V0', '0.000126';
   requires 'Test2::Harness', '0.001099';
+  requires 'Test::More', '0.88';
```
Comment (daxelrod-datto) [inline cpanfile]: I think that this test will in the future be trivial to port to Test2::V0 but no reason to block on this at the moment.

File: scripts/tests/findlang.pl [hunk 2]
```diff
@@ -48,13 +48,11 @@
 while (my $line = <$h>) {
     chomp($line);
     $line =~ /svn/ and next;
-    my ($file) = $line =~ / ([^ ]+)$/;
-    my ($lang) = $line =~ /%lang\(([^\)]+)\)/;
-    $lang ||= '';
+    my @parts = split(' ', $line);
+    my $file = $parts[-1];
```
Comment (daxelrod-datto) [inline scripts/tests/findlang.pl]: This is so much clearer (and more correct) than the previous code.

**Comment by Conan-Kudo** [comment]
@daxelrod-datto This should be ready to go now. :smile:

**Pull request merged**